src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import java.util.ArrayList;
+import java.io.IOException;
+
+/**
+ * TaskList class to store all the tasks in Duke.
+ */
+public class TaskList {
+    private ArrayList<Task> items;
+    Storage storage;
+
+    /**
+     * Constructor for TaskList class.
+     */
+    public TaskList() {
+        this.storage = new Storage();
+        this.items = storage.readData();
+    }
+
+    /**
+     * Exit function for TaskList class, writes data to .txt file.
+     */
+    public void bye() {
```

Comment:
Bye might be a little confusing haha. Consider including the fact that it writes data.